src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -6,46 +6,49 @@
   >
     <v-radio
       id="radio-24"
-      class="radio-button pt-2"
+      class="pt-2"
+      color="#1976d2"
```

Comment:
We should not have hex colours in code.

Should this be "primary" instead?

If this is a very special case (but I didn't see it in Leo's PR with radio buttons), since it isn't easy to add a new colour to both this repo and the apps that use this component, I'd be OK with using `:color="myNamedColor"` here and declaring myNamedColor in the code below.

Reply:
Yes I had to add this specifically as the buttons were grey colored. I will change as you suggested

Reply:
Will the radio buttons in other components also need fixing?

Does this work correctly if this component is imported into one of our UIs (in which we declare a theme color "primary")?

I am not yet convinced that this is a special enough case to locally declare the color value (instead of using the base CSS variables).

Reply:
Looks like this will be an issue for other components as well. I have Leo's PR running locally and it has the same issue.
DEV:
![image](https://github.com/bcgov/bcrs-shared-components/assets/116035339/e3799e45-aa7e-45cc-acdb-97108e996ff3)

LOCAL:
![image](https://github.com/bcgov/bcrs-shared-components/assets/116035339/5de312a9-cb19-4596-88f1-7d4f52f71642)

Reply:
What do you get if you use a color variable name here, like "primary"?

Reply:
I tried with `color="primary"` and it works